template.yml

```
@@ -196,6 +199,99 @@ Resources:
         Comment: "Check request authentication and rewrite requests"
         Runtime: cloudfront-js-1.0
 
+  InvalidationFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: index.handler
+      Runtime: "nodejs14.x"
```

```suggestion
      Runtime: "nodejs16.x"
```
userei l'ultimo runtime disponibile se non è un problema

---

Fatto!